### ISSUE TITLE
[FEATURE] Display workflows a component is connected to in "Components" tab

### ISSUE THREAD

**Issue by LucBerge** [PipedreamHQ/pipedream#8154]
**Is your feature request related to a problem? Please describe.**
I have a lot of published components.
Some of them are used daily on prod workflows in my company.
I want to easily see which custom component is used in which workflow.

**Describe the solution you'd like**
Just like I can see the workflows a source is connected to, I would like to see the workflows a component is connected to.

**Do you have a workaround?**
I have to go to each workflow and manually verify each action.

**Comparable features in other tools?**
I don't know

**Additional context**
Linked to

**Comment by dannyroosevelt** [comment]
100% agree! There is a lot we can do to improve component management overall (manage updates and push them out to all deployed instances for example, in addition to being able to author components, etc).

Are there additional suggestions or ideas you have for improving that experience?

**Comment by LucBerge** [comment]
> Are there additional suggestions or ideas you have for improving that experience?

Yes, there is something you can do in order to create more actions, faster and without github accounts. But you will need a new ticket. This is something you probably already ready have on the roadmap:

---------------

There are a lot of non implemented actions in all the APIs. If a user wants to use a new endpoint, he has 2 possibilities:

1. Fork, Code, PR
2. Select "Use any Wix API in Node js" and implement

The second option is the fastest but other users does not benefit from it.
You must have a process to automatically add the custom user component to Gtihub.

Example:
- User create a new "Use any Wix API in Node js" for a new endpoint
- After workflow deployment and 20 calls, this component is marked as candidate to PR.
- Based on the user code, you can ask Pi to create a PR adding the user component (which we already know works) to the Github repo
- Someone review the code and fix issues if needed


In order to check whether this is relevant, you must have a KPI to know how many people use "Custom API in Node" vs "Already implemented actions".
